hack/windows/Clean-AntreaNetwork.ps1

```diff
@@ -19,53 +25,27 @@ $OVS_DB_SCHEMA_PATH = "$OVSInstallDir\usr\share\openvswitch\vswitch.ovsschema"
 $OVSDB_CONF_DIR = "$OVSInstallDir\etc\openvswitch"
 $OVS_DB_PATH = "$OVSDB_CONF_DIR\conf.db"
 $OVS_BR_ADAPTER = "br-int"
+$AntreaHnsNetworkName = "antrea-hnsnetwork"
 
-function GetHnsnetworkId($NetName) {
-    $NetList= $(Get-HnsNetwork -ErrorAction SilentlyContinue)
-    if ($NetList -eq $null) {
-        return $null
-    }
-    foreach ($Net in $NetList) {
-        if ($Net.Name -eq $NetName) {
-            return $Net.Id
-        }
-    }
-    return $null
-}
-
-function ClearHyperVBinding($adapter) {
-    $status= $(Get-NetAdapterBinding -Name $adapter.Name -ComponentID vms_pp).Enabled
-    if ($status -EQ "False") {
-        Set-NetAdapterBinding -Name $adapter.Name -ComponentID vms_pp -Enabled $False
-    }
-}
-
-function ClearHyperVBindingOnAdapter($adapterName) {
-    if ($adapterName -NE "") {
-        $adapter = $(Get-NetAdapter -Name $adapterName)
-        if ($adapter -eq $null) {
-            return
-        }
-        ClearHyperVBinding($adapter)
-    } else {
-        $adapters= $(Get-NetAdapter | ? Virtual -EQ $false)
-        if ($adapters -eq $null) {
-            Write-Host "Physical network adapters not found"
-            return
-        }
-        foreach ($adapter in $adapters) {
-            ClearHyperVBinding($adapter)
-        }
-    }
+function RemoveOVSService() {
+    stop-service ovs-vswitchd
+    sc.exe delete ovs-vswitchd
+    stop-service ovsdb-server
+    sc.exe delete ovsdb-server
```

Comment:
will this fail if one of the services has already been stopped or deleted?

Reply:
It would not fail if the service is already stopped. For a nonexist service ( deleted ), yes, it may fail, but it does not block the following commands